operator/config/crd/bases/forklift.konveyor.io_ovirtvolumepopulators.yaml

```diff
@@ -39,10 +39,13 @@ spec:
                 type: string
               engineUrl:
                 type: string
+              imageName:
```

Comment:
ah so it's still here?

Reply:
forgotten :)

Reply:
hold your horses! it doesn't sounds so bad..

Reply:
dropped